ddtrace/profiling/exporter/_packages.py

```diff
@@ -90,30 +91,28 @@ def _build_package_file_mapping():
     return mapping
 
 
-class _load_status(enum.Enum):
-    FAILED = "failed"
-
-
-_FILE_PACKAGE_MAPPING = None  # type: typing.Optional[typing.Union[typing.Dict[str, Distribution], _load_status]]
+_FILE_PACKAGE_MAPPING = None  # type: typing.Optional[typing.Dict[str, Distribution]]
+if asbool(os.getenv("DD_PROFILING_ENABLE_CODE_PROVENANCE", False)):
+    # DEV: If code provenance is enabled, create the mapping as soon as this
+    # module is imported. This should happen in the parent process to
+    # guarantee that the work is not repeated in child worker processes.
+    try:
+        _FILE_PACKAGE_MAPPING = _build_package_file_mapping()
+    except Exception:
```

Comment:
Is `Exception` used because there's a really wide range of exceptions that could occur here?

Reply:
Yes, I suppose we don't really know what could go wrong here, so we catch any errors and report them.